docs/topics/testing/tools.txt

```diff
@@ -555,6 +581,11 @@ The only exceptions that are not visible to the test client are
 exceptions internally and converts them into the appropriate HTTP response
 codes. In these cases, you can check ``response.status_code`` in your test.
 
+If ``Client.raise_request_exception`` is ``False``, the test client will return a
+500 response as would be returned to a browser. The response has the attribute
+:attr:`~Response.exc_info` to provide information about the unhandled
```

Comment:
Normally the name of the attribute precedes the word attribute, i.e. "exc_info attribute".